includes/ph-formatting-functions.php

```diff
@@ -19,7 +19,10 @@
  * @param string|array $var
  * @return string|array
  */
-function ph_clean( $var ) {
+function ph_clean( $var, $is_phone_number = false ) {
```

Comment:
As discussed. i think this might be better being a separate function. Only because ph_clean is used in thousands of places throughout the app so to create a separate parameter for 3-4 occurrences seems a bit overkill :)